namespacing.php

```diff
@@ -0,0 +1,19 @@
+<?php 
+	
+namespace Foo_Bar;
+
+class Plugin {
+	public $config = array();
+	
+	public function __construct()
+	{
```

Comment:
Braces need to be fixed. Also, you've got some strange indentation happening - I think you've mixed tabs and spaces. Make sure it's all tabs.